tests/functional/spec/interstitial.spec.js

```diff
@@ -17,6 +17,7 @@ describe('Interstitial Pages', function () {
 
         // When
         expect(InterstitialPage.getMainHeading()).to.contain('Breakfast Interstitial Page')
```

Comment:
Modifying it in that way removes any expectation on button content and relevance to this PR. You want it reverted?